donut/modules/newsgroups/routes.py

```diff
@@ -178,4 +179,16 @@ def posting_positions(group_id):
     if 'username' not in flask.session:
         return flask.abort(403)
     user_id = auth_utils.get_user_id(flask.session['username'])
-    return flask.jsonify(helpers.get_posting_positions(group_id, user_id))
+    if auth_utils.is_admin():
+        pos = [{
+            'pos_id': groups.get_position_id('Devteam', 'Devteam Member'),
+            'pos_name': 'Devteam Member'
+        }]
+    pos.extend(helpers.get_posting_positions(group_id, user_id))
+    return flask.jsonify(pos)
+
+
+def get_user_actions(user_id):
+    if auth_utils.is_admin():
+        return {'send': True, 'control': True}
```

Comment:
This denies the `receive` permission if the user is an admin. Will that prevent admins from viewing messages sent to their newsgroups?

Reply:
Anyone who is a member can view the group. If `receive` is true, the page will display that the user is subscribed which can be false for admins

Reply:
Oh hmm it seems like that might be a bug that positions without `receive` can still see newsgroup messages, but that doesn't need to be addressed now